common/src/main/scala/hmda/model/filing/ParserValidValuesLookup.scala

```diff
@@ -24,8 +28,22 @@ object ParserValidValuesLookup {
   val parserValidValuesLines = fileLines(s"/$parserValidValuesFile")
   val parserValidValuesMap = parserValidValuesMapCreator(parserValidValuesLines)
 
-  def lookupParserValidValues(fieldName: String): String =
+  val parserValidValuesLines2024 = fileLines(s"/$parserValidValuesFile2024")
+  val parserValidValuesMap2024 = parserValidValuesMapCreator(parserValidValuesLines2024)
+
+  def lookupParserValidValues(fieldName: String): String = {
+
     parserValidValuesMap
       .getOrElse(fieldName, "")
+  }
+
+
+  def lookupParserValidValuesByYear(fieldName: String,period: Period): String =
+    period match {
+
+      case Period(2024, Some(_)) => parserValidValuesMap2024.getOrElse(fieldName, "")
+      case Period(2024, None)    => parserValidValuesMap2024.getOrElse(fieldName, "")
```

Comment:
I see that we had the `(FIG 2024)` suffix in the original txt file, why do the new file route instead of sticking with that?
also, I'm guessing it's not just 2024, right? it's 2024 and onwards? so it probably should be:

```scala
period match {
  case Period(yr, _) if yr < 2024 => {pre 2024 description}
  case _ => {2024 description}
}
```
and if a new one comes out after that, it'd look like:
```scala
period match {
  case Period(yr, _) if yr < 2024 => {pre 2024 description}
  case Period(yr, _) if yr < 2027 => {pre 2027 description}
  case _ => {2027 description}
}
```